skyportal/handlers/api/group.py

```diff
@@ -547,6 +547,98 @@ def delete(self, group_id, username):
         return self.success()
 
 
+class GroupUsersFromOtherGroupsHandler(BaseHandler):
+    @auth_or_token
+    def post(self, group_id, *ignored_args):
+        """
+        ---
+        description: Add users from other group(s) to specified group
+        parameters:
+          - in: path
+            name: group_id
+            required: true
+            schema:
+              type: integer
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  fromGroupIDs:
+                    type: array
+                    items:
+                      type: integer
+                    type: boolean
+                required:
+                  - fromGroupIDs
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+        """
+        try:
+            group_id = int(group_id)
+        except (TypeError, ValueError):
+            return self.error("Invalid group_id parameter: must be an integer")
+
+        if not has_admin_access_for_group(self.associated_user_object, group_id):
+            return self.error("Requesting user is not an admin of this group.")
+
+        data = self.get_json()
+
+        from_group_ids = data.get("fromGroupIDs")
+        if from_group_ids is None:
+            return self.error("Missing required parameter: fromGroupIDs")
+        if not isinstance(from_group_ids, (list, tuple)):
+            return self.error(
+                "Improperly formatted fromGroupIDs parameter; "
+                "must be an array of integers."
+            )
+        group = DBSession().query(Group).get(group_id)
+        from_groups = (
+            DBSession().query(Group).filter(Group.id.in_(from_group_ids)).all()
+        )
+        user_ids = set()
+        for group in from_groups:
+            for user in group.users:
+                # Ensure user has sufficient stream access to be added to group
+                if group.streams and "System admin" not in user.permissions:
```

Comment:
Perhaps we should define an `accessible_streams` parameter simllar to `accessible_groups`

Reply:
Then check that rather than the "System admin" ACL explicitly

Reply:
Will defer to a later PR - this pattern is used in several places